app-guides/livebook.html.markerb

```diff
@@ -0,0 +1,120 @@
+fly launch --no-deploy
+```
+
+Follow the prompts. Respond `Y` to copy the configuration from your existing `fly.toml` to the new app. You'll be asked for an app name and a primary region, which will be added to your local `fly.toml`. The app name must be unique across Fly.io. The primary region is the region where Machines are created for this app when not otherwise specified.
+
+## Set a Livebook password
+
+Livebook stores your password in an environment variable. Set that variable securely using `fly secrets set`.
+
+```cmd
+fly secrets set LIVEBOOK_PASSWORD="<my-super-sekrit-password>"
+```
+
+Don't use this example password as a real password.
+
+## Deploy the app
+
+Now that the app has been created and the `LIVEBOOK_PASSWORD` secret is set, you can deploy to a Machine.
+
+The `fly deploy` command creates a new release of an app using the `fly.toml` in your working directory, and rolls that out to its Machines. 
+
+The first time it's run on a given app, and when it's run on an app that currently doesn't have any Machines, `fly deploy` does some extra work. In our case, it provisions public IP addresses for the HTTP service and creates the volume referred to in the `mounts` section in `fly.toml`, before creating and starting the app's Machine(s).
+
+```
+fly deploy --vm-size shared-cpu-2x --volume-initial-size 1 --ha=false
+```
+
+The `fly deploy` command accepts some options to tune the initial resource provisioning.
+
+Livebook seems to want something a bit heftier than the default `shared-cpu-1x` Machine with 256MB RAM, so for the purpose of demonstration we chose a `shared-cpu-2x` size, which comes with 512MB. You can experiment with CPU and memory to see what you need for your project.
+
+We set the size of the volume to be created explicitly, although 1GB happens to be the default size for a new volume.
+
+The `--ha=false` option tells `fly deploy` not to [create a second Machine for redundancy](/docs/reference/app-availability/#redundancy-by-default-on-first-deploy). Fly volumes don't automatically synchronize throughout an app, so the simplest way to run this app is as a single Machine, with the availability repercussions that entails.
+
+<div class="note icon">
+At the time of writing, you may see this error message as deployment finishes: 
+<code>WARNING The app is not listening on the expected address and will not be reachable by fly-proxy.</code><br>
+<code>You can fix this by configuring your app to listen on the following addresses:</code><br>
+<code>  - 0.0.0.0:8080</code>
+
+If you've followed these instructions thoroughly, you can ignore this message. It's a false alarm caused by provisioning of the initial resources causing the startup time to take longer than flyctl expects. 
+</div>
+
+## Visit your Livebook app
+
+To use your livebook, visit `https://<your-app-name>.fly.dev` in the browser, or type `fly apps open`, and enter your password where prompted. Create a new notebook, and save it in the `/data` directory. 
```

Comment:
Curious how you do this specifically when `/data` seems to already be the default for disk?

Reply:
There's a file browser in the Livebook.

![image](https://github.com/superfly/docs/assets/95245363/02eaec7a-8c8c-4de1-97f6-a4492c5b0469)

You can go up a dir and save it in /app for example, where it wouldn't be persisted.